README.md

```diff
@@ -64,8 +64,10 @@ Option | Description
 ---    | ---
 `--latest` | Get the `latest` versions of all components, referring to their `develop` branches.
 `--no-pleuston` | Start up Ocean without an instance of `pleuston`. Helpful for development on `pleuston`.
-`--local-parity-node` | Runs a local parity POA node and Secret Store instead of ganache-cli.
+`--local-secret-store` | Runs a local parity POA node and Secret Store instead of ganache-cli.
 `--reuse-database` | Start up Ocean and reuse the Database from ganache. Helpful for development.
```

Comment:
should be something like `--reuse-ganache-database` because it triggers a reuse of the blockchain db not the mongodb. This is also a super special case.